result.json

```diff
@@ -0,0 +1,15775 @@
+
+{
+    "address_components": [
+        {
+            "long_name": "375",
+            "short_name": "375",
+            "types": [
+                "street_number"
+            ]
+        },
+        {
+            "long_name": "11th St",
+            "short_name": "11th St",
+            "types": [
+                "route"
+            ]
+        },
+        {
+            "long_name": "South of Market",
+            "short_name": "South of Market",
+            "types": [
+                "neighborhood",
+                "political"
+            ]
+        },
+        {
+            "long_name": "San Francisco",
+            "short_name": "SF",
+            "types": [
+                "locality",
+                "political"
+            ]
+        },
+        {
+            "long_name": "San Francisco County",
+            "short_name": "San Francisco County",
+            "types": [
+                "administrative_area_level_2",
+                "political"
+            ]
+        },
+        {
+            "long_name": "California",
+            "short_name": "CA",
+            "types": [
+                "administrative_area_level_1",
+                "political"
+            ]
+        },
+        {
+            "long_name": "United States",
+            "short_name": "US",
+            "types": [
+                "country",
+                "political"
+            ]
+        },
+        {
+            "long_name": "94103",
+            "short_name": "94103",
+            "types": [
+                "postal_code"
+            ]
+        }
+    ],
+    "adr_address": "<span class=\"street-address\">375 11th St</span>, <span class=\"locality\">San Francisco</span>, <span class=\"region\">CA</span> <span class=\"postal-code\">94103</span>, <span class=\"country-name\">USA</span>",
+    "formatted_address": "375 11th St, San Francisco, CA 94103, USA",
+    "formatted_phone_number": "(415) 626-1409",
+    "geometry": {
```

Comment:
What is the viewport??? Why does it have two location?

Reply:
where is viewport?

Reply:
I think there are partial of address and total address. 
We need formatted_address